### PR TITLE
bugs and lint fixes in flip-flops

### PR DESCRIPTION
**Pull request by Dishebh**
#### Changes done:
- [x] Bugs and lint fixes in flip-flops

#### Preview Link(s): https://deploy-preview-394--tender-ardinghelli-ebb79f.netlify.app/docs/flipflop/sr_flipflop.html
https://deploy-preview-394--tender-ardinghelli-ebb79f.netlify.app/docs/flipflop/jk_flipflop.html
https://deploy-preview-394--tender-ardinghelli-ebb79f.netlify.app/docs/flipflop/d_flipflop.html
https://deploy-preview-394--tender-ardinghelli-ebb79f.netlify.app/docs/flipflop/t_flipflop.html
https://deploy-preview-394--tender-ardinghelli-ebb79f.netlify.app/docs/flipflop

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

File: docs/flipflop/jk_flipflop.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ JK flip-flop is the modified version of SR flip-flop. It operates with only posi
 
 This circuit has two inputs J & K and two outputs Q(t) & Q(t)’. The operation of JK flip-flop is similar to SR flip-flop. Here, the inputs of SR flip-flop are considered as **S = J Q(t)’** and **R = KQ(t)** in order to utilize the modified SR flip-flop for 4 combinations of inputs.
 
-## State table of **SR** flip-flop
+## State table of JK flip-flop
```
Comment (Shivansh2407) [inline docs/flipflop/jk_flipflop.md:24]: SR

File: docs/flipflop/t_flipflop.md [hunk 2]
```diff
@@ -47,9 +47,8 @@ From the above characteristic table, the next state equation can be directly wri
 
 
 ```yaml
-          Q(t+1)=T′Q(t)+TQ(t)′
-           
-          ⇒  Q(t+1)=T⊕Q(t)
+          Q(t +1 ) = T'.Q(t) + T.Q(t)'
```
Comment (Shivansh2407) [inline docs/flipflop/t_flipflop.md:50]: Q(t + 1) = T'.Q(t) + T.Q(t)'
Remove space after 1 and add a space before it as I have done

File: docs/flipflop/jk_flipflop.md (hunk 1 above)
Comment (Dishebh) [reply docs/flipflop/jk_flipflop.md:24]: But the state table is of JK, and it is the `jk_flipflop` section.

File: docs/flipflop/t_flipflop.md (hunk 2 above)
Comment (Dishebh) [reply docs/flipflop/t_flipflop.md:50]: Sure, thanks :+1:

File: docs/flipflop/jk_flipflop.md (hunk 1 above)
Comment (Shivansh2407) [reply docs/flipflop/jk_flipflop.md:24]: Oh my bad sweat_smile.

**Pull request merged**